.drone.star

```diff
@@ -79,19 +55,7 @@ config = {
                 "webUIcliTwoFactorTOTP": "webUIcliTOTP",
             },
             "phpVersions": [
-                "7.2",
                 "7.3",
-            ],
-        },
-        "webUIother-74": {
-            "suites": {
-                "webUIapiTwoFactorTOTP": "webUIapiTOTP",
-                "webUIcliTwoFactorTOTP": "webUIcliTOTP",
-            },
-            "servers": [
-                "daily-master-qa",
-            ],
-            "phpVersions": [
```

Comment:
this as well?

Reply:
Same as above - actually more PHP 7,4 pipelines should run.